setup.cfg

```diff
@@ -6,7 +6,7 @@
 # E241: multiple spaces after ','
 # E501: line too long (xx > 79 characters)
 # W503: line break occurred before a binary operator
-ignore = E127,E128,E129,E226,E241,E501,W503
+ignore = E127,E128,E129,E226,E241,E501,W503,N801,N802,N806
```

Comment:
`N802` and `N806` make sense to me as we sometimes need to follow Qt's camel case style.

But could we have "N801: class names should use CapWords convention", or are there important exceptions to this rule in our code base?

Reply:
Those are for flake8 and enabling them cause a lot of noise, I'd keep them as is for now. But feel free to experiment.